File: LeetCode/Easy/2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.py
class Solution:
    def leftRightDifference(self, nums: List[int]) -> List[int]:
        lst=[]
        for i in range(len(nums)):
            ans=abs(sum(nums[:i])-(sum(nums[i:])-nums[i]))
            lst.append(ans)
        return lst

        

File: LeetCode/Easy/2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.cpp
class Solution {
public:
    vector<int> leftRightDifference(vector<int>& nums) {
        vector<int> lst;

        for (int i = 0; i < nums.size(); i++) {
            int leftSum = accumulate(nums.begin(), nums.begin() + i, 0);
            int rightSum = accumulate(nums.begin() + i, nums.end(), 0) - nums[i];

            int ans = abs(leftSum - rightSum);
            lst.push_back(ans);
        }

        return lst;
    }
};